### ISSUE TITLE
Improve clarity around key and region

### ISSUE THREAD

**Issue by jimbobbennett** [jimbobbennett/SpeechToTextSamples#4]
To reduce confusion, there should be more clarity about how the key and region are loaded from the environment variables.

**Comment by naseeb0** [comment]
Yes, that's true. I was too confused at first. If someone wants to try out the code quickly without making the .env file then they can add the key and region manually in quoted form but rather if someone wants to run the code in organized form then they can follow the Readme to better understand how to use the code.
ThankYou
